chore: [CHAOS-2338]: Added docs for Chaos Module release notes - 1.17.3 (#2962)
harness/developer-hub: release-notes/chaos-engineering.md

Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
:::

## Latest - August 21, 2023, version 1.17.3

<Tabs>
  <TabItem value="What's new">

* Added support for CSMs to be able to view the Activity and Usage for Chaos Module. (CHAOS-1978)

> * Added support for Openshift configuration for deploying chaos infrastructure. This will provide users with a predefined SCC & will allow users to modify the same according to their needs. (CHAOS-1889)

Author: Done!

> * Enhanced the Chaos Experiment execution diagram to not switch to running nodes automatically. This will allow users to stay on any node by clicking on the same & observe the respective node details. (CHAOS-2258)

Author: Done!

> * Enhanced docker service kill fault to support containerd runtime. (CHAOS-2220)

Author: Done!

> * Added support for targetting applications by only appkind, only applabel and set-based labels. (CHAOS-2170) and (CHAOS-2128)

Author: Done!

> * Parallel chaos injection/revert operations at scale have been improved for multiple target pods on the same node. (CHAOS-1563)

Author: Done!

> * Updated the behaviour of TARGET_CONTAINER configuration. when not set, the fault will target all containers in target pods. Previously, it used to target only one container randomly. (CHAOS-1216)

Author: Done!

* Now, Users can specify drain timeout explicitly in the node drain fault. The node-drain fault has been using the CHAOS_DURATION value as a timeout, leading to potential confusion and risk of failure, especially when a shorter duration is used with many pods. The expectation is that CHAOS_DURATION should define the unschedulable period after draining. Providing a specific drain timeout would help users better estimate the eviction time for all pods on a node, reducing errors and false negatives. (CHAOS-2185)

* Enhanced the JobCleanUpPolicy configuration to also retain helper pods when it is set to retain in ChaosEngine. (CHAOS-2273)

  </TabItem>
  <TabItem value="Early access">

This release does not include early access features.

  </TabItem>
  <TabItem value="Fixed issues"> 

> * Fixed revert chaos for Node drain chaos fault in case of Abort or Chaos Injection failure. (CHAOS-2184)

Reviewer: Not clear. Can we write it this way?
```suggestion
* Fixed how chaos is reverted if an attempt to inject the node drain fault fails or needs to be canceled. (CHAOS-2184)
```
The [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/abort-abortion) says "Use abort for a technical audience only if it's part of the application programming interface (API). Use an alternative term in comments and descriptions." It lists some alternative terms for comments and descriptions. Can we use cancel instead? At least in descriptive documentation like this one?

Author: Done!
